manager/manager_test.go

```diff
@@ -400,3 +400,110 @@ func TestManagerLockUnlock(t *testing.T) {
 	// error.
 	<-done
 }
+
+// If the root CA materia is updated in the memory store, a manager will update its own
```

Comment:
I appreciate it!  I tend to misspell a bunch of stuff.  I wonder if we should add `github.com/client9/misspell/cmd/misspell` to the linter?